.changeset/happy-cups-design.md

```diff
@@ -0,0 +1,9 @@
+---
+"@comet/brevo-api": major
+---
+
+Refactor `email-campaign` and `target-group` entity
+
+Use `createEmailCampaignEntity` for creating `email-campaign` entity. Pass `EmailCampaignContentBlock`, `Scope` and `TargetGroup`.
+
+Use `createTargetGroupEntity` for creating `target-group` entity. Pass `Scope` and optional `BrevoFilterAttributes`
```

Comment:
I would add the information that they need to be passed to the module's config

Reply:
066b5a59ab0e57f6a1764580089b4a3b689e585d